### ISSUE TITLE
Problem with serializing models with default_scope

### ISSUE THREAD

**Issue by MartinElvar** [DockYard/postgres_ext-serializers#3]
Hallow,

If i try to serialize a model, which have a default scope i get an error.

````
default_scope { order('name ASC') }
````

````
PG::GroupingError - ERROR:  column "services.name" must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: ...ds_ids"."id" FROM "service_fields_ids")  ORDER BY name ASC),...
````

**Comment by felixbuenemann** [comment]
This will be fixed once PR #45 is merged. It solved the problem by moving the order clause into the aggregate function.

**Comment by felixbuenemann** [comment]
Support for order on default and custom `has_many` scopes was added in 89c3ff02e3d24b6d3b4e039061809add55016aee and merged in PR #45.

If you still have issues, open a new report.
